Ska/engarchive/fetch.py

```diff
@@ -485,6 +515,10 @@ def _get_msid_data_cached(content, tstart, tstop, msid, unit_system):
         files and cache recent results.  Caching is very beneficial for derived
         parameter updates but not desirable for normal fetch usage."""
         return MSID._get_msid_data(content, tstart, tstop, msid, unit_system)
+        #global _backend
```

Comment:
Until MAUDE has solid support for stats just raise an exception here if the `maude` backend is requested.